public/worldrender/worldschema.h

```diff
@@ -0,0 +1,78 @@
+#ifndef WORLD_SCHEMA_H
+#define WORLD_SCHEMA_H
+
+#ifdef COMPILER_MSVC
+#pragma once
+#endif
+
+#include <tier0/platform.h>
+#include <tier1/utlstring.h>
+#include <tier1/utlvector.h>
+#include <mathlib/vector.h>
+#include <mathlib/vector2d.h>
+
+// See tier0/basetypes.h :
+//   AlliedModders: We don't need this, and it conflicts with generated proto headers
+#ifndef schema
+#define schema
+#endif
```

Comment:
I don't think we need any schema decls at all, since we anyway don't have anything to use it for